content/v2.2/app/container-and-components.md

```diff
@@ -333,11 +333,17 @@ require "hanami"
 
 module Bookshelf
   class App < Hanami::App
-    config.no_auto_register_paths << "structs"
+    config.no_auto_register_paths << "values"
   end
 end
 ```
 
+Note that the default value for `no_auto_register_paths` is `["db", "entities", "relations", "structs"]`,
+so you do not need to specify those. Also, be sure to append to that list.
+
+These apply for the root of the application, as well within any slices.
```

Comment:
I think we should link to slices here. If a reader is going through the docs in rough order of appearance, they won't have got to slices yet.

Reply:
Good call, added!

This makes me think we should consider re-ordering the guides, as well (perhaps on the new site). The current order is: Booting, Container and components, Providers, Settings, Autoloading, Environments, App config, Inflector, Code reloading, Slices.

Slices are more core than code reloading, or the inflector IMO. I think Autoloading and Environments could be lower as well. Perhaps: Booting, Container and components, Providers, Settings, App config, Slices, Autoloading, Environments, Inflector, Code reloading?

I still think linking to Slices is a good idea regardless of the order.

Reply:
Thanks @cllns :) Agree that it'd be worth trying a different order. We came up with this order years ago for 2.0 and haven't reconsidered it since. I think an "Overview" page for the App guide would be useful thing to have too. That way we can introduce all the key concepts in relation to each other, and provide relevant jumping off links to learn more.